httsleep/main.py

```diff
@@ -34,6 +34,9 @@ class HttSleeper(object):
                      function that takes the response as an argument returning True.
     :param auth: a (username, password) tuple for HTTP authentication.
     :param headers: a dict of HTTP headers.
+    :param verify: (optional) Either a boolean, in which case it controls whether we verify
```

Comment:
At the moment, the other kwargs (e.g. `auth` and `headers` and so on) aren't marked with "(optional)". It'd be best to stick to that style and I'll take care of them all at once. 

(Unless you'd like to change them all :) I agree, marking it explicitly in the documentation is better.)

Reply:
How we handle this probably depends on the outcome of the other discussion thread, on the whole of this description being copied from the `requests` library.  So I'll wait until you've had a chance to respond to that before I decide which way to go!